tracing-subscriber/src/fmt/format/json.rs

```diff
@@ -76,6 +188,8 @@ where
         let meta = event.metadata();
 
         let flatten_event = self.format.flatten_event;
+        let disable_current_span = self.format.disable_current_span;
+        let disable_span_list = self.format.disable_span_list;
```

Comment:
Why are these `let` bindings needed?

Reply:
they are not, it was how it was already done: https://github.com/tokio-rs/tracing/blob/master/tracing-subscriber/src/fmt/format/json.rs#L78.
I removed them